### PR TITLE
Doc/readme

### PR DESCRIPTION
**Pull request by ivyhungtw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+To properly use the app and login feature, make sure you have filled out the following information in .env file.
+
+You can register your own IMGUR client id on [IMGUR](https://api.imgur.com/oauth2/addclient).
+
+```
+JWT_SECRET=<your_jwt_secret>
+IMGUR_CLIENT_ID=<your_imgur_client_id>
+```
+
```
Comment (sherryliao21) [inline README.md:112]: 這邊我們要不要到時候上傳一個 .env.example 檔？還是直接叫使用者自己建？

File: README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+# Simple Twitter API
+
+A RESTful API server, built with Node.js, Express, and mySQL, for [Simple Twitter](https://github.com/ivyhungtw/simple-twitter) project.
```
Comment (sherryliao21) [inline README.md]: 可修改成

> A RESTful API server for [Simple Twitter](https://github.com/ivyhungtw/simple-twitter) project built with Node.js, Express framework, and mySQL.

一開始就告訴別人這 API 對應的是什麼東西，後面再詳述技術

File: README.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+All URLs referenced in the API documentation have the following host and base path:
+
+```
+https://simple-twitter-api-2021.herokuapp.com/api
+```
+
+## API Guideline
+
+Please refer to [API Docs](https://simple-twitter-api-2021.herokuapp.com/api-docs/) for more details.
```
Comment (sherryliao21) [inline README.md]: > Please refer to [API Docs](https://simple-twitter-api-2021.herokuapp.com/api-docs/) for more details.
> You can also test these APIs with the Swagger UI with the account below.
> 
> Admin
> Account: root
> Password: 12345678
> 
> User
> Account: user1
> Password: 12345678

我們的 swagger 頁面到時候 perform all actions will be required an access token ，所以需要提供這兩個使用者帳密給他們登入才能操作模擬

File: README.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+#### Reply related
+
+| Method | Path                      | description            | req.body         | res.json tables |
+| ------ | ------------------------- | ---------------------- | ---------------- | --------------- |
+| POST   | /tweets/:tweet_id/replies | 新增回覆               | comment (text>0) | --------        |
+| GET    | /tweets/:tweet_id/replies | 瀏覽一則推文的所有回覆 | --------         | replies, users  |
+
+## Install Simple Twitter API
+
+Following the instruction, you can run a Simple Twitter API server on your local machine.
```
Comment (sherryliao21) [inline README.md]: Following the instruction, you can run a Simple Twitter API server on your local machine.

> By* following the instruction, you can run a Simple Twitter API server on your local machine.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+#### Prerequisites
+
+- [Git](https://git-scm.com/downloads)
+- [Node.js v14.15.1](https://nodejs.org/en/download/)
+- [MySQL v8.0.20](https://dev.mysql.com/downloads/mysql/)
+- [MySQL Workbench v8.0.20](https://dev.mysql.com/downloads/mysql/)
+
+#### Clone the repository locally
```
Comment (sherryliao21) [inline README.md]: #### Clone the repository locally

> #### Clone the repository *to your local machine

File: README.md (hunk 1 above)
Comment (ivyhungtw) [reply README.md:112]: 感覺都可以

**Review by sherryliao21 — APPROVED** [review]
Approved 😎

**Pull request merged**